### ISSUE TITLE
Check Point #2 is Ready for Review

### ISSUE THREAD

**Issue by arice-student** [arice-student/LabPal#1]
@pawaitemadisoncollege - Check Point 2 is completed and is ready for review.

**Comment by pawaitemadisoncollege** [comment]
Nice job working through all the issues this week to hit checkpoint 2! I know you had frustrating day or two; your hard work shows very clearly here! 

- [ ] I'm curious what errors were introduced into the unit tests when you added Lombok?  This should work, and my guess is that it's something small and easy to fix (such as ignoring one side of the one to many in the toString and equals methods).
- [ ] Some markdown formatting could be improved. Here, I think just adding a space after the hashes will do the trick: 
![Screen Shot 2021-03-08 at 10 14 18 AM](https://user-images.githubusercontent.com/13874466/110348320-23405380-7ff7-11eb-8a43-72387a330012.png)
- [ ] Replace the printlns and stack traces in the database class with log statements. 
![Screen Shot 2021-03-08 at 10 11 17 AM](https://user-images.githubusercontent.com/13874466/110348370-32270600-7ff7-11eb-81db-157401d0aa6d.png)
- [ ] If you are feeling at all overwhelmed with the number of tables and entities, you could put protocol, specification, and method as stretch goals. We could talk about other ways to reduce complexity too - I want to make sure this is sized in a way that makes you comfortable! 
- [ ] Time to start building out that project plan beyond week 9. Note that your time on the indie project will likely be less in weeks 9-12 due to the team project.

**Comment by arice-student** [comment]
- I will plan on using Lombok again when I setup my other entity classes and if the error messages come up again I will send them to the class in Slack.
- I just went ahead and looked through my student files (journal, project plan, time log, weekly reflection, etc.) and added a space to the markdown headers where it was needed, so everything should be good-to-go now.
- It looks like I had the log messages in the Database.java file, however, I forgot to remove the printStackTrace() methods.  I just removed those from the file, so everything should be good-to-go there.
- I think I will need a least a method for my application, however, I will plan on removing the protocol and specification and may add those in later if I have time.
- My last day of work is today, so I plan to spend a significant amount of time this week to catch-up and hopefully get ahead in the class.
<img width="742" alt="Screen Shot 2021-03-16 at 8 43 51 AM" src="https://user-images.githubusercontent.com/70419798/111319866-6ff9df00-8634-11eb-8eee-99fa9f8362be.png">
